playbooks/provisioning/openstack/sample-inventory/group_vars/all.yml

```diff
@@ -0,0 +1,42 @@
+env_id: "openshift"
+openstack_dns_domain: "example.com"
+openstack_nameservers: ["192.168.1.1"]
```

Comment:
That's not necessary, and could potentially even cause problems with a circular resolution. With PR https://github.com/redhat-cop/casl-ansible/pull/48, the DNS name server config is being consolidated and named `public_dns_nameservers` as a more appropriate name - i.e.: what's the "fall-back" DNS server(s) in case the internal ones don't resolve the requested name.